templates/default/enclave.config.yaml

```
@@ -3,26 +3,26 @@ chains:
       bonding_registry:
         address: "0x8A791620dd6260079BF849Dc5567aDC3F2FdC318"
-        deploy_block: 13
+        deploy_block: 12
       slashing_manager:
         address: "0x5FC8d32690cc91D4c39d9d3abcBD16989F875707"
         deploy_block: 11
       fee_token:
         address: "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512"
-        deploy_block: 6
+        deploy_block: 5
       e3_program:
```

YAMLlint (1.38.0)

[error] duplication of key "e3_program" in mapping

(key-duplicates)